spec/features/admin_claim_approve_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.feature "Admin approves a claim" do
+  context "User is logged in as a service operator" do
+    before do
+      stub_dfe_sign_in_with_role(AdminSession::SERVICE_OPERATOR_DFE_SIGN_IN_ROLE_CODE)
+      visit admin_path
+      click_on "Sign in"
+    end
+
+    scenario "User can approve a claim" do
+      freeze_time do
+        submitted_claims = create_list(:claim, 5, :submitted, submitted_at: DateTime.now, approved_at: nil)
```

Comment:
You shouldn't need to set `submitted_at` and `approved_at` if you're using the `submitted` trait:

```suggestion
        submitted_claims = create_list(:claim, 5, :submitted)
```